website/src/controllers/payments/paypalBillingSetup.js

```diff
@@ -1,3 +1,5 @@
+require('babel-core/register');
```

Comment:
Could you explain why babel-core/register shouldn't be used?  As I understand it, it's discouraged for production code because of how slow it is, but this particular script should never appear in production (if it does, that seems problematic).

Anyways, if you don't want that line, I'll take it out and replace it with some notes that might help out others that run into the same issues I did.